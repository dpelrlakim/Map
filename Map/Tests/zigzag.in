[] a 10
[] a 20
[] a 11
[] a 19
[] a 12
[] a 18
erase a 10
erase a 20
erase a 11
erase a 19
erase a 12
erase a 18
