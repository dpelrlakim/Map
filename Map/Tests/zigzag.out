0
a: (10,0)
0
a: (10,0) (20,0)
0
a: (10,0) (11,0) (20,0)
0
a: (10,0) (11,0) (19,0) (20,0)
0
a: (10,0) (11,0) (12,0) (19,0) (20,0)
0
a: (10,0) (11,0) (12,0) (18,0) (19,0) (20,0)
a: (11,0) (12,0) (18,0) (19,0) (20,0)
a: (11,0) (12,0) (18,0) (19,0)
a: (12,0) (18,0) (19,0)
a: (12,0) (18,0)
a: (18,0)
a:
